build/scripts/build-runtime.sh

```diff
@@ -17,10 +17,8 @@ cp -R "$DIST_DIR/TKLiveSync.framework" "$INTERNAL_DIR"
 
 . "$WORKSPACE/build/scripts/build.sh"
 
-mkdir -p "$INTERNAL_DIR/NativeScript/Frameworks"
-
-cp -R "$DIST_DIR/NativeScript/" "$INTERNAL_DIR/NativeScript"
-cp -R "$DIST_DIR/NativeScript.framework" "$INTERNAL_DIR/NativeScript/Frameworks"
+mkdir -p "$INTERNAL_DIR/NativeScript"
+cp -R "$DIST_DIR/NativeScript.framework" "$INTERNAL_DIR/NativeScript/NativeScript.framework"
```

Comment:
Let's drop the `NativeScript` framework now that it contains only the framework, what do you think?